.travis.yml

```diff
@@ -24,9 +24,13 @@ jobs:
         - mvn clean test
     - stage: deploy-scipooldev
       script:
+        - chmod +x ./pomSnapshotToGitHash.sh
```

Comment:
git preserves file attributes so if you commit these bash scripts with `chmod +x` then you don't need this step.